<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="ru.testqa.novotelecom.addressbook.tests.ContactDeletionTests" tests="2" skipped="0" failures="1" errors="0" timestamp="2018-03-13T03:41:38" hostname="nt-07" time="4.666">
  <properties/>
  <testcase name="testContactDeletion" classname="ru.testqa.novotelecom.addressbook.tests.ContactDeletionTests" time="2.41">
    <failure message="java.lang.AssertionError: &#10;Expected: &lt;[ContactData{id='202', firstname='firstname 1', lastname='lastname 1'}, ContactData{id='201', firstname='firstname 0', lastname='lastname 0'}, ContactData{id='200', firstname='firstname 2', lastname='lastname 2'}, ContactData{id='199', firstname='firstname 1', lastname='lastname 1'}, ContactData{id='198', firstname='firstname 0', lastname='lastname 0'}, ContactData{id='197', firstname='firstname 2', lastname='lastname 2'}, ContactData{id='196', firstname='firstname 1', lastname='lastname 1'}, ContactData{id='195', firstname='firstname 0', lastname='lastname 0'}]&gt;&#10;     but: was &lt;[ContactData{id='203', firstname='firstname 2', lastname='lastname 2'}, ContactData{id='202', firstname='firstname 1', lastname='lastname 1'}, ContactData{id='201', firstname='firstname 0', lastname='lastname 0'}, ContactData{id='200', firstname='firstname 2', lastname='lastname 2'}, ContactData{id='199', firstname='firstname 1', lastname='lastname 1'}, ContactData{id='198', firstname='firstname 0', lastname='lastname 0'}, ContactData{id='197', firstname='firstname 2', lastname='lastname 2'}, ContactData{id='196', firstname='firstname 1', lastname='lastname 1'}]&gt;" type="java.lang.AssertionError">java.lang.AssertionError: 
Expected: &lt;[ContactData{id='202', firstname='firstname 1', lastname='lastname 1'}, ContactData{id='201', firstname='firstname 0', lastname='lastname 0'}, ContactData{id='200', firstname='firstname 2', lastname='lastname 2'}, ContactData{id='199', firstname='firstname 1', lastname='lastname 1'}, ContactData{id='198', firstname='firstname 0', lastname='lastname 0'}, ContactData{id='197', firstname='firstname 2', lastname='lastname 2'}, ContactData{id='196', firstname='firstname 1', lastname='lastname 1'}, ContactData{id='195', firstname='firstname 0', lastname='lastname 0'}]&gt;
     but: was &lt;[ContactData{id='203', firstname='firstname 2', lastname='lastname 2'}, ContactData{id='202', firstname='firstname 1', lastname='lastname 1'}, ContactData{id='201', firstname='firstname 0', lastname='lastname 0'}, ContactData{id='200', firstname='firstname 2', lastname='lastname 2'}, ContactData{id='199', firstname='firstname 1', lastname='lastname 1'}, ContactData{id='198', firstname='firstname 0', lastname='lastname 0'}, ContactData{id='197', firstname='firstname 2', lastname='lastname 2'}, ContactData{id='196', firstname='firstname 1', lastname='lastname 1'}]&gt;
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18)
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:6)
	at ru.testqa.novotelecom.addressbook.tests.ContactDeletionTests.testContactDeletion(ContactDeletionTests.java:32)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:648)
	at org.testng.TestRunner.run(TestRunner.java:505)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)
	at org.testng.TestNG.runSuites(TestNG.java:1028)
	at org.testng.TestNG.run(TestNG.java:996)
	at org.gradle.api.internal.tasks.testing.testng.TestNGTestClassProcessor.runTests(TestNGTestClassProcessor.java:136)
	at org.gradle.api.internal.tasks.testing.testng.TestNGTestClassProcessor.stop(TestNGTestClassProcessor.java:80)
	at org.gradle.api.internal.tasks.testing.SuiteTestClassProcessor.stop(SuiteTestClassProcessor.java:59)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.gradle.messaging.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:35)
	at org.gradle.messaging.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.messaging.dispatch.ContextClassLoaderDispatch.dispatch(ContextClassLoaderDispatch.java:32)
	at org.gradle.messaging.dispatch.ProxyDispatchAdapter$DispatchingInvocationHandler.invoke(ProxyDispatchAdapter.java:93)
	at com.sun.proxy.$Proxy2.stop(Unknown Source)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.stop(TestWorker.java:116)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.gradle.messaging.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:35)
	at org.gradle.messaging.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.messaging.remote.internal.hub.MessageHub$Handler.run(MessageHub.java:360)
	at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:54)
	at org.gradle.internal.concurrent.StoppableExecutorImpl$1.run(StoppableExecutorImpl.java:40)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
</failure>
  </testcase>
  <testcase name="testContactDeletionAlert" classname="ru.testqa.novotelecom.addressbook.tests.ContactDeletionTests" time="2.242"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
